tests/DynamicObject.Tests/Main.fs

```diff
@@ -4,7 +4,7 @@ open Fable.Pyxpecto
 
 let all = testSequenced <| testList "DynamicObj" [
     ReflectionUtils.Tests.main
-    DynamicObj.Tests.main
+    DynamicObjs.Tests.main
```

Comment:
Why the "s"?

Reply:
that was the name of the file i refactored into separate ones 🤷

Reply:
ah also, this: 
>  A namespace and a module named 'DynamicObj.Tests' both occur in two parts of this assembly